.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+# Name of the workflow
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master and stable branches
+on:
+  push:
+    branches: [ master, stable ]
+  pull_request:
+    branches: [ master, stable ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # Main building and testing
+  build-test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Define the platforms to run on. Later determines the Docker image that'll be used
+    strategy:
+      matrix:
+        platform: [ c7-dev-py2, c7-dev, c8-dev, u18-dev, u20-dev ]
+
+    # Sets up environment variables
+    env:
+      PLATFORM: ${{ matrix.platform }}
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out our repository under $GITHUB_WORKSPACE, so our job can access it
+    - uses: actions/checkout@v2
+
+    # Pulls the associated Docker image
+    - name: Docker pull
+      run: |
+        if [ "$PLATFORM" == "c7-dev-py2" ];
+        then
+          docker pull hepsoftwarefoundation/c7-dev
+        else
+          docker pull hepsoftwarefoundation/$PLATFORM
+        fi
+
+    # Builds the code and runs the test
+    - name: Build and Test
+      run: |
+        if [ "$PLATFORM" == "c7-dev-py2" ];
+        then
+          docker run -e CMAKE=cmake3 -e CMAKE_EXTRA=-DPYTHON_TEST=python2 -v $(pwd):/mnt hepsoftwarefoundation/c7-dev scl enable devtoolset-8 /mnt/.github/scripts/build-test.sh;
```

Comment:
Update to use $DIMAGE, if you agree with the above suggestion

Reply:
Done.